py3status/modules/sysdata.py

```diff
@@ -271,15 +298,32 @@ def sysData(self):
             self.values['cpu_temp'] = cpu_temp
             self.py3.threshold_get_color(cpu_temp, 'temp')
 
-        # get RAM usage info
+        # get RAM/SWAP usage info
+        memi = self.data.mem(
+            self.mem_unit,
+            self.swap_unit,
+            self.py3.format_contains(self.format, 'mem_*'),
+            self.py3.format_contains(self.format, 'swap_*')
```

Comment:
why not just do something like  `self.swap_info = self.py3.format_contains(self.format, 'swap_*')` etc in `post_config_hook()` then it's easily available anywhere you need it.